api/lib/domain/models/Assessment.js

```diff
@@ -145,6 +145,10 @@ class Assessment {
     return challenges[Math.abs(hashInt(this.id)) % challenges.length];
   }
 
+  get hasRecordedFocusedout() {
```

Comment:
```suggestion
  get hasRecordedFocusedOut() {
```

Reply:
Je ne vois pas de diff :sweat: 

Pour ma part, le nom est un peu bizarre.
L'intention est  "sur cette question, l'utilisateur est sorti de la zone autorisée"
Le sujet de `recorded` est quoi ? qui ? L'API ? Le front ?

Je propose
```suggestion
  get hasBeenFocusedOut() {
```

Reply:
Effectivement c'est plus parlant